### PR TITLE
JP-2106: Update keyword schema enum lists

### LINKED ISSUE #6078: Update keyword enum lists in datamodels schemas
_Issue [JP-2106](https://jira.stsci.edu/browse/JP-2106) was created on JIRA by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse):_

Some keywords have had new entries added to their enum lists in the KWD for B7.8. Our datamodels schemas need to be updated to stay in synch. Specifics for each keyword are given in comments below.

### PR DESCRIPTION
**Pull request by hbushouse**
Closes #6078 

Resolves [JP-2106](https://jira.stsci.edu/browse/JP-2106)

**Description**
Update the enum lists for a few keywords to bring them into synch with what's in the KWD for B7.8

Checklist
- [ ] Tests
- [ ] ~~Documentation~~
- [x] Change log
- [x] Milestone
- [x] Label(s)

### CODE REVIEW

File: jwst/datamodels/schemas/core.schema.yaml [hunk 1]
```diff
@@ -246,10 +246,12 @@ properties:
             enum: [OK, SUSPECT]
             fits_keyword: ENG_QUAL
             blend_table: True
-          pointing_engdb_quality:
+          engdb_pointing_quality:
```
Comment (tapastro) [inline jwst/datamodels/schemas/core.schema.yaml:249]: Will this change break calls like https://github.com/spacetelescope/jwst/blob/20d5bbbaeb6c1f25fb23bf12550c01be10c881f7/jwst/lib/set_telescope_pointing.py#L467 ? I only found references to 'pointing_engdb_quality' in set_telescope_pointing, at least.

Comment (hbushouse) [reply jwst/datamodels/schemas/core.schema.yaml:249]: D'oh! Yes, it sure would break those. Updating the script for the next commit. Good catch!

**Review by tapastro — APPROVED** [review]
LGTM!

**Comment by hbushouse** [comment]
Current CI failures are due to CRDS server being off-line for OS patching.

**Review by stscieisenhamer — APPROVED** [review]

**Pull request merged**